src/Blog/Archive/ArchiveRepository.php

```diff
@@ -95,7 +95,7 @@ private function extractFromDateColumn(string $attr): Fragment
             $str = ['year' => '%Y', 'month' => '%m', 'day' => '%d'][$attr];
             return new Fragment("strftime('{$str}', post.published_at) {$attr}");
         }
-        return new Fragment("extract({$attr} from post.published_at) {$attr}");
+        return new Fragment("extract({$attr} from post.published_at) \"{$attr}\"");
```

Comment:
How about to quote the second usage $attr?

Reply:
The first {$attr} has a fixed set of values and is internal for SQL.